code/src/app.css

```diff
@@ -0,0 +1,78 @@
+body {
+  background-color: black;
+  color: white;
+}
+
+a {
+  color: white;
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.album-cards {
+  position: relative;
+}
+.image-container {
+  position: relative;
+}
+
+.image-container:hover {
```

Comment:
The hover effects look weird in some widths as well:

<img width="1001" alt="Skärmavbild 2020-10-26 kl  11 30 36" src="https://user-images.githubusercontent.com/51529201/97161882-ad8dc680-177e-11eb-81e3-305193347ac5.png">